handsontable/src/renderers/numericRenderer/numericRenderer.js

```diff
@@ -49,6 +49,8 @@ export function numericRenderer(instance, TD, row, col, prop, value, cellPropert
     }
 
     cellProperties.className = classArr.join(' ');
+
+    TD.dir = 'ltr';
```

Comment:
The TD elements that are injected by Walkontable are reused during the HoT usage. To make sure that the TD attributes do not go over to another cell (after CRUD operations, hiding columns/rows, etc) the TD attribute has to be cleared.

May I suggest updating [the line of code](https://github.com/handsontable/handsontable/blob/feature/issue-8791/handsontable/src/3rdparty/walkontable/src/renderer/cells.js#L92) and add `TD.removeAttribute('dir');`?

Reply:
Resolved within 92a351e